client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -90,6 +95,8 @@ class SiteTools extends Component {
 		const exportText = translate(
 			'Export content from your site. You own your data — take it anywhere!'
 		);
+		const cloneTitle = translate( 'Clone' );
```

Comment:
`clone` can be a verb and a noun - can we add a `context: 'verb'` attribute here on the translate call to make it clear?